tests/foreman/ui/test_containerimagetag.py

```diff
@@ -44,9 +48,11 @@ def module_repository(module_product):
         url=CONTAINER_REGISTRY_HUB,
     ).create()
     repo.sync()
-    return repo
+    yield repo
+    repo.delete()
```

Comment:
Deleting anything just waits time given we do not keep the Satellite after testing.

Reply:
What would you suggest @spusateri do instead of cleaning up entities if he's seeing test instability if this test module is executed more than once?

Reply:
yes random names?

Reply:
Hi, I have confirmed that we are using random names here by running multiple times and viewing within the Satellite. Since we are using the entities `EntitiesCreateMixin`, and name is a require value, a random name will be assigned when create_missing() is called even though I am not passing a specific random name as with `url` or `content_type`.

Reply:
I think this indicates a bit more failure analysis is necessary to identify what's flaky and actually failing. If there's a conflict in satellite with this repository type, this cleanup should absolutely be included. Otherwise, there may be an entirely different path and pattern we need to address.